### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by d4straub**
This is for release 1.2.0, changes in the CHANGELOG.md!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,25 +40,44 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
...
+By default, the pipeline currently performs the following:
+
+* Sequencing quality control ([FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
+* Trimming of reads ([Cutadapt](https://journal.embnet.org/index.php/embnetjournal/article/view/200))
+* Illumina read processing with [QIIME2](https://www.nature.com/articles/s41587-019-0209-9)
+* Infer Amplicon Sequence Variants (ASVs) ([DADA2](https://doi.org/10.1038/nmeth.3869))
+* Taxonomical classification based on [SILVA](https://www.arb-silva.de/) [v132](https://www.arb-silva.de/documentation/release-132/) or [UNITE](https://unite.ut.ee/) database
+* excludes unwanted taxa, produces absolute and relative feature/taxa count tables and plots, plots alpha rarefaction curves, computes alpha and beta diversity indices and plots thereof ([QIIME2](https://www.nature.com/articles/s41587-019-0209-9))
+* Calls differentially abundant taxa ([ANCOM](https://www.ncbi.nlm.nih.gov/pubmed/26028277))
+* Overall pipeline run summaries (`MultiQC`)
```
Comment (emnilsson) [inline README.md:56]: Now MultiQC does not point to a publication or to a source for the software, is this intended?

Also, further up under the introduction it says "currently supported is Illumina paired end data", maybe this should be changed to include that PacBio is supported?

Comment (d4straub) [reply README.md:56]: Good points, have addressed them in  #217, please approve that linked PR so that the changes appear here.

**Review by emnilsson — APPROVED** [review]
I've looked through the code and looked at the results from some of the tests, and it looks good. Great job!

**Review by jtangrot — APPROVED** [review]
Looks good to me! I've looked through the changes and done some testing, and all seems OK.

**Comment by d4straub** [comment]
Thanks!

**Pull request merged**